src/utils/math/translate.js

```diff
@@ -0,0 +1,17 @@
+export default function translate({ viewHandler, direction, percent, rtl }) {
```

Comment:
Hmm.. I don't think this filename and function name is properly describing what it is doing. If so, what is it translating?

I see it updates the dataView of the viewHandler... Also I don't think that is a typical "util" file then, so I suggest we rename it into something more describing and move it somewhere else than the utils folder, maybe into the viewHandler?

Reply:
good idea!

Reply:
fixed!